cloudflare/package.json

```
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "deploy": "npx wrangler deploy --minify",
-    "publish": "npx wrangler publish --minify",
-    "dev": "npx wrangler dev --port 19194 --remote"
+    "test-debug": "npx wrangler dev --port 19194 --remote --env debug",
+    "test-release": "npx wrangler dev --port 19194 --remote"
```

```suggestion
    "dev-debug": "npx wrangler dev --port 19194 --env debug",
    "dev": "npx wrangler dev --port 19194"
```

This is not a test command so let's not use `test`. The additional params such as `--remote` can be passed anyways with `npm run` so can drop those too.